client/my-sites/upgrades/domain-management/dns/dns-record.jsx

```diff
@@ -79,17 +81,17 @@ var DnsRecord = React.createClass( {
 	getName: function() {
 		const { name, service, protocol, type } = this.props.dnsRecord,
 			domain = this.props.selectedDomainName,
-			isRoot = name === domain + '.';
+			isRoot = name === `${ domain }.`;
 
 		if ( 'SRV' === type ) {
-			return service + '_' + protocol + '.' + ( isRoot ? name + '.' : '' ) + domain;
+			return `_${ service }._${ protocol }.${ isRoot ? '' : name + '.' }${ domain }`;
```

Comment:
That's putting it lightly...
Before it looked like this:
<img width="753" alt="screen shot 2016-07-08 at 13 14 36" src="https://cloud.githubusercontent.com/assets/3392497/16685838/172a3f94-450e-11e6-8adf-c0ba0fa0d8d9.png">

Now, it's:
<img width="729" alt="screen shot 2016-07-08 at 13 18 55" src="https://cloud.githubusercontent.com/assets/3392497/16685903/8fca79d2-450e-11e6-8726-1c32018cef8d.png">
Note that there was no `.` between service and protocol, too 😀 
(and, yes, that leading underscore is required for SRV records - that way you can just copy and paste it to `dig`)